fabric-network/src/impl/event/fulltransactioneventfactory.ts

```diff
@@ -41,6 +41,7 @@ export function newFullTransactionEvent(blockEvent: BlockEvent, txEnvelopeIndex:
 
 		const envelope: any = block.data.data[txEnvelopeIndex];
 		const transactionId = envelope.payload.header.channel_header.tx_id;
+		const timestamp = envelope.payload.header.channel_header.timestamp;
```

Comment:
We should be able to fluff up a Date object from the string with `new Date(envelope.payload.header.channel_header.timestamp)`